public/index.html

```diff
@@ -88,6 +88,8 @@
     <link rel="stylesheet" href="fonts.css" type="text/css" />
     <script>
       window.EXCALIDRAW_ASSET_PATH = "/";
+      // setting this so when libraries installation happens in same window.
```

Comment:
```suggestion
      // setting this so that libraries installation reuses this window tab.
```

just to make it clearer